cardea-server/src/main/java/ca/on/oicr/gsi/cardea/server/service/CaseService.java

```diff
@@ -147,7 +147,8 @@ public TreeSet<ShesmuDetailedCase> getShesmuDetailedCases() {
 
   private Set<ShesmuSequencing> getSequencingForShesmuCase(Case kase) {
 
-    Set<ShesmuSequencing> sequencings = new HashSet<>();
+    TreeSet<ShesmuSequencing> sequencings =  new TreeSet<>(Comparator.comparing(ShesmuSequencing::getTest).thenComparing(shesmuSequencing -> String.join(shesmuSequencing.getLimsIds().toString())));
```

Comment:
You're only passing one parameter to `String#join` - the result of `TreeSet#toString`. The first parameter is the delimiter, and you're not providing any items to join, so this won't return anything. It's compiling because the second parameter is varargs, so omitting it is allowed (though this is rarely wanted). You'll need to provide a delimiter and not call `toString` on the LIMS IDs